girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js

```diff
@@ -85,11 +85,23 @@ var GeojsImageViewerWidgetExtension = function (viewer) {
             const xOffset = transformInfo.xoffset || 0;
             const yOffset = transformInfo.yoffset || 0;
             const matrix = transformInfo.matrix || [[1, 0], [0, 1]];
-            if (xOffset === 0 && yOffset === 0 && matrix === [[1, 0], [0, 1]]) {
-                return '+proj=longlat +axis=enu';
+            const s11 = matrix[0][0];
+            const s12 = matrix[0][1];
+            const s21 = matrix[1][0];
+            const s22 = matrix[1][1];
+
+            let projString = '+proj=longlat +axis=enu';
+            if (xOffset > 0) {
```

Comment:
Is there a reason we constrain the offsets to be positive?

Reply:
In #678, `xOffset` and `yOffset` had a minimum of `0`. In rethinking that decision, I can't come up with a truly compelling reason for that constraint. There's likely a valid reason to render an overlay with its origin outside of the base image. To that end, I've pushed 86cc293